src/bepasty/tests/test_app.py

```diff
@@ -2,11 +2,29 @@
 # app tests
 #
 
+from flask import request, url_for
+from flask.views import MethodView
+
+import pytest
+
 from ..app import create_app
 from ..config import Config
 
 
-def test_secret_key():
+@pytest.fixture
+def app_fixture():
+    app_base_path = Config.APP_BASE_PATH
+    permissions = Config.PERMISSIONS
+    secret_key = Config.SECRET_KEY
+
+    yield
+
+    Config.APP_BASE_PATH = app_base_path
+    Config.PERMISSIONS = permissions
+    Config.SECRET_KEY = secret_key
```

Comment:
did you check if you can use `monkeypatch` instead of this?

Reply:
looks like works, fixed